apps/fabric-website/src/SiteDefinition/SiteDefinition.tsx

```diff
@@ -0,0 +1,60 @@
+  customizations: FluentCustomizations,
+  platforms: {
+    web: {
+      name: 'Web',
+      icon: 'TVMonitor'
+    },
+    ios: {
+      name: 'iOS',
+      icon: 'AppleLogo-platformPicker'
+    },
+    android: {
+      name: 'Android',
+      icon: 'AndroidLogo-platformPicker'
+    }
+  },
+  pages: [
+    {
+      title: 'Fabric',
+      url: '#/',
+      className: 'fabricPage',
+      isHomePage: true,
+      isUhfLink: true,
+      isContentFullBleed: true,
+      component: () => <LoadingComponent title="Welcome to Microsoft UI Fabric" />,
+      getComponent: cb => require.ensure([], require => cb(require<any>('../pages/HomePage/HomePage').HomePage))
+    },
+    StylesPages,
+    ControlsPages,
+    ResourcesPages,
+    AboutPages,
+    {
+      title: 'Demo Loading Page',
+      url: '#/ms-loading',
+      isHiddenFromMainNav: true,
+      component: () => <LoadingComponent title="Demo Loading Page" />
+    },
+    {
+      title: 'Template Page',
+      url: '#/ms-page-template',
+      isHiddenFromMainNav: true,
+      component: () => <LoadingComponent title="Template Page" />,
+      getComponent: cb => require.ensure([], require => cb(require<any>('../pages/PageTemplates/TemplatePage/TemplatePage').TemplatePage))
+    }
+  ],
+  redirects: {
+    '#/customizations/': '#/controls/web/customizations/',
```

Comment:
I think you guys are working together, but I think @natalieethell 's #8844 modifies this--just flagging for a future update